Lib/logging/config.py

```diff
@@ -33,6 +33,7 @@
 import sys
 import threading
 import traceback
+from inspect import signature
```

Comment:
IIRC importing `inspect` can be expensive, so consider only importing it if and where needed. If it *is* kept here, note that it needs to go in the above list in alphabetical order (I know other parts of `logging` don't follow that, but this module does). We may not even need it - see comment below.